chroot-packages.list

```diff
@@ -32,7 +32,7 @@ bzip2, ca-certificates, curl, gzip, less, sudo
 python3-jsonschema, reprepro, python3-apt
 
 # uploading
-python-crcmod, python-pip, python-setuptools, python-wheel, ssh
+python-crcmod, python-pip, python-setuptools, python-wheel, rsync, ssh
```

Comment:
This file doesn't seem to repeat dependencies that were already listed earlier. If this is a convention we're following, then this is redundant.

Reply:
imo we should list all dependencies required for each task, since the way we accomplish each task might change